tt_metal/hw/ckernels/wormhole_b0/metal/llk_api/llk_sfpu/ckernel_sfpu_bitwise_or.h

```diff
@@ -0,0 +1,28 @@
+// SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+#pragma once
+#include "ckernel.h"
+#include "ckernel_defs.h"
+#include "noc_nonblocking_api.h"
+using namespace sfpi;
+namespace ckernel {
+namespace sfpu {
+template <bool APPROXIMATION_MODE, int ITERATIONS = 8>
+inline void calculate_bitwise_or(const uint value) {
+#pragma GCC unroll 0
+    for (int d = 0; d < ITERATIONS; d++) {
+        vInt input = dst_reg[0];
+        vInt res = reinterpret<vInt>(input | value);
```

Comment:
Why is a `reinterpret<vInt>` needed here? OR output will be vInt?

Should the input values be vUint, and reinterpreted back to vInt for the sign handling after the OR? Similar to the bitwise and operation above?

Reply:
`reinterpret<vInt>` is Not needed.  I will remove the reinterpret in bitwise_and also because it works as same with or without it since we are implementing the op for positive tensors only.